config/defaults.go

```diff
@@ -114,18 +106,18 @@ func getNewRelicDefaults() *NewRelicConfig {
 	}
 }
 
-func getNodesDefaults() *NodesConfig {
+func getNodesDefaults() *ARCConfig {
```

Comment:
I would rename this function to getArcDefalts or something 🤔

Reply:
Right 👍